src/app/submission/sections/upload/file/section-upload-file.component.html

```diff
@@ -1,8 +1,21 @@
 <ng-container *ngIf="fileData">
   <div class="row">
-    <div class="col-md-12">
+    <!-- Default switch -->
+    <div class="col-md-2 d-flex justify-content-center align-items-center" >
+      <div class="custom-control custom-switch">
+        <input 
+        type="checkbox" 
+        class="custom-control-input" 
+        id="primaryBitstream{{fileIndex}}" 
+        [disabled]="processingSaveStatus$ | async"
+        [checked]="isPrimary" 
+        (change)="togglePrimaryBitstream($event)">
+        <label class="custom-control-label" for="primaryBitstream{{fileIndex}}"></label>
+      </div>
+    </div>
+    <div class="col-md-10">
       <div class="float-left w-75">
-        <h3>{{fileName}} <span class="text-muted">({{fileData?.sizeBytes | dsFileSize}})</span></h3>
+        <h3>{{fileName}}<span class="text-muted">({{fileData?.sizeBytes | dsFileSize}})</span></h3>
```

Comment:
It looks like you accidentally removed the space between the filename and the file size.  I don't think we should do that.